test/e2e/utils/test_context.go

```diff
@@ -88,27 +88,36 @@ func (kc *KBTestContext) InstallCertManager() error {
 	if _, err := kc.Kubectl.Command("create", "namespace", "cert-manager"); err != nil {
 		return err
 	}
-	_, err := kc.Kubectl.Apply(false, "-f", fmt.Sprintf("https://github.com/jetstack/cert-manager/releases/download/%s/cert-manager.yaml", certmanagerVersion), "--validate=false")
+	_, err := kc.Kubectl.Apply(false, "-f",
+		fmt.Sprintf("https://github.com/jetstack/cert-manager/releases/download/%s/cert-manager.yaml",
+			certmanagerVersion), "--validate=false")
```

Comment:
Could we fit this in two lines?

Reply:
It would be more readable to construct the file path in a variable above `kc.Kubectl.Apply()` and pass it as an arg such that all arguments are on one line.

Reply:
Changed.